src/sentry/sentry_metrics/metrics_interface.py

```diff
@@ -0,0 +1,42 @@
+from abc import ABC, abstractmethod
+from typing import Mapping, Sequence, Union
+
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
+
+
+class GenericMetricsBackend(ABC):
+    @abstractmethod
+    def counter(
+        self,
+        use_case_id: UseCaseID,
+        org_id: int,
+        project_id: int,
+        metric_name: str,
+        value: Union[int, float],
+        tags: Mapping[str, str],
+    ) -> None:
+        raise NotImplementedError()
+
+    @abstractmethod
+    def set(
+        self,
+        use_case_id: UseCaseID,
+        org_id: int,
+        project_id: int,
+        metric_name: str,
+        value: Sequence[Union[int, str]],
```

Comment:
Are you sure we support strings in the indexer protocol and in clickhouse here ?

Reply:
we don't. I think Lyn was asking about the `set` datatype and there was some confusion around that.